apps/clabs/routes.py

```diff
@@ -0,0 +1,269 @@
+# -*- encoding: utf-8 -*-
+import os
+from flask import render_template, request, jsonify, current_app
+from flask_login import current_user
+from datetime import datetime, timedelta
+from .store import load_state, save_state, prune_inplace, default_state_path
+from . import blueprint
+from .mock_data import CLABS, CLABS_DETAILS
+
+# Tentamos usar PyYAML; se não estiver instalado, seguimos sem quebrar
+try:
+    import yaml  # PyYAML
+except Exception:
+    yaml = None
```

Comment:
if we need pyaml we should add it to the requirements, instead of this try/except. Please review this part